test_scripts/Resumption/InteriorVehicleData/024_InteriorVD_double_subscription_with_moduleId.lua

```diff
@@ -37,13 +36,13 @@ common.Step("App registration", common.registerAppWOPTU)
 common.Step("App activation", common.activateApp)
 for _, moduleType in pairs(common.modules)do
   common.Step("Subscription to " .. moduleType, common.GetInteriorVehicleData,
-    { moduleType, common.getModuleId(moduleType, moduleIdNumber), isSubscribed, isNotCached, expectNotif })
+    { moduleType, common.getModuleId(moduleType, 2), isSubscribed, isNotCached, expectNotif })
```

Comment:
according to this update, maybe `WARNINGS` resultCode can be used directly in the script without local constant.

Reply:
Done (see 1442497  )